src/test/java/io/jenkins/plugins/analysis/core/scm/BlameFactoryTest.java

```diff
@@ -0,0 +1,85 @@
+package io.jenkins.plugins.analysis.core.scm;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.plugins.git.GitSCM;
+import hudson.scm.SCM;
+
+import io.jenkins.plugins.analysis.core.util.JenkinsFacade;
+
+import static org.mockito.Mockito.*;
+
+class BlameFactoryTest {
+
+    private JenkinsFacade jenkinsFacadeMock = mock(JenkinsFacade.class);
+    private Run runMock = mock(Run.class);
+    private GitChecker gitCheckerMock = mock(GitChecker.class);
+
+    @AfterEach
+    void resetBlameFactory() {
+        BlameFactory.setGitChecker(new GitChecker());
+        BlameFactory.setJenkinsFacade(new JenkinsFacade());
+    }
+
+    @Test
+    void shouldCreateNullBlamerOnMissingGitPlugin() {
+        when(jenkinsFacadeMock.isPluginInstalled("git")).thenReturn(false);
+        BlameFactory.setJenkinsFacade(jenkinsFacadeMock);
```

Comment:
Rather than using a field and these two lines of code in the test, use a method, e.g.
```
enableGitPlugin(true/false);
```

This method registers the facade stub in the `BlameFactory`.